samples/serial/socat-daemon.service

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=service that uses socat to relay /dev/ttyVirtS0 to network connection
+After=network-online.target
+
+[Service]
+ExecStart=/usr/bin/socat pty,link=/dev/ttyVirtS0,raw,user=iotedge-user,group=dialout,mode=777 tcp:172.18.246.137:5002
```

Comment:
777 permissions should be avoided, given we have control over the user & group inline here, the permission mask could be restricted

Reply:
change access mode to 660